pkg/orchestrator/etcd_worker_test.go

```diff
@@ -272,7 +272,8 @@ func (s *etcdWorkerSuite) TestEtcdSum(c *check.C) {
 	}
 
 	err = errg.Wait()
-	if err != nil && (errors.Cause(err) == context.DeadlineExceeded || errors.Cause(err) == context.Canceled) {
+	if err != nil && (errors.Cause(err) == context.DeadlineExceeded || errors.Cause(err) == context.Canceled ||
```

Comment:
Please keep line width within 80 chars
```suggestion
	if err != nil && (errors.Cause(err) == context.DeadlineExceeded || 
		errors.Cause(err) == context.Canceled ||
		strings.Contains(err.Error(), "etcdserver: request timeout")) {
```